psydac/api/ast/utilities.py

```diff
@@ -221,16 +252,7 @@ def compute_atoms_expr_field(atomic_exprs, indices_quad,
         updates += [AugAssign(val,'+',Mul(*args))]
         # ...
 
-        # ... map basis function
-
-        if mapping and  isinstance(atom, _partial_derivatives):
-            name       = print_expression(atom)
-            rhs        = LogicalExpr(mapping, atom)
-            rhs        = SymbolicExpr(rhs)
-            map_stmts += [Assign(Symbol(name), rhs)]
-    # ...
-
-    return inits, updates, map_stmts
+    return inits, updates, map_stmts, atomic_exprs
 
 #=============================================================================
 #def compute_atoms_expr_field(atom, indices_quad,
```

Comment:
@saidctb I think we can remove this old code now. Thanks for the improvements!

Reply:
yes I have removed now .